include/fmt/chrono.h

```diff
@@ -430,6 +430,45 @@ auto write(OutputIt out, const std::tm& time, const std::locale& loc,
   return write_encoded_tm_str(out, string_view(buf.data(), buf.size()), loc);
 }
 
+#if FMT_SAFE_DURATION_CAST
```

Comment:
This can be moved into `fmt_duration_cast`.

Reply:
Done, if I understood what you meant correctly!